### DISCUSSION TITLE
Possible to get the path inside a policy?

### DISCUSSION THREAD

**Discussion by fatfrog** [varvet/pundit#714] [Q & A]
This is an odd request I know, but is it possible to get the path to the endpoint being authorized in Pundit, like so?
```
class Admin::PicturesPolicy < ApplicationPolicy
  def index?
    @user.permissions.include?(<---- current_path to pictures#index ------>)
  end

**Comment by Burgestrand** [comment]
Have you seen [Additional Context](https://github.com/varvet/pundit#additional-context)?
